src/Panels/Panels.js

```diff
@@ -1049,7 +1049,9 @@ module.exports = kind(
 		}
 		var orig = ev.originator,
 			idx = this.getPanelIndex(orig),
-			target = this.getSpotlightTarget('LEFT', orig);
+			target = this.getSpotlightTarget('LEFT', orig),
+			active = this.getActive(),
+			h = active ? active.getHeader() : null;
```

Comment:
I guess I was thinking we could do something like the following instead, but this is more of a stylistic thing and I don't feel super strongly:
```
h = active && active.getHeader();
```